charts/prometheus-snmp-exporter/Chart.yaml

```diff
@@ -15,3 +15,5 @@ maintainers:
     email: maxime@root314.com
   - name: xiu
     email: github@xiu.io
+  - name: tuc
```

Comment:
If you have an interest in becoming a maintainer for the chart, get yourself known in #1656. We can't amend this list as you're not a maintainer yet.